web/templates/group/form.html.eex

```diff
@@ -0,0 +1,20 @@
+<%= form_for @changeset, @action, fn f -> %>
+  <%= if @changeset.action do %>
+    <div class="alert alert-danger">
+      <p>Oops, something went wrong! Please check the errors below.</p>
+    </div>
+  <% end %>
+
+  <%= hidden_input f, :owner_id, value: value_from_changeset(@changeset, :owner_id) %>
+  <%= error_tag f, :owner_id %>
```

Comment:
Basically shouldn't ever be used, but just in case something terribly wonky happens, it'd be nice for the foreign key constraint error (or others having to do with owner) to have somewhere to go that is visible to the user.